core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/common.scss

```diff
@@ -0,0 +1,49 @@
+@use "variables" as *;
+@import "mixins";
+
+#input-text-base {
```

Comment:
Concern that this common can get out of hand really quick, let's keep an weye on it.

Reply:
this should be only used for styles that are shared, for component specific styles we should use the respective file